charts/plugin-br-bank-transfer-jd/values.yaml

```
@@ -0,0 +1,615 @@
+      cpu: 250m
+      memory: 512Mi
+    limits:
+      cpu: "1"
+      memory: 1Gi
+
+  podSecurityContext:
+    runAsUser: 1001
+    runAsGroup: 1001
+    fsGroup: 1001
+    fsGroupChangePolicy: "OnRootMismatch"
+    seccompProfile: { type: RuntimeDefault }
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)